docker-compose.yml

```diff
@@ -37,17 +37,18 @@ services:
     command:
       - -c
       - |
-        HASH=$$(tor --hash-password "$${TOR_AUTH_PASSWORD:-coinswap}" | grep "^16:")
+        HASH=$$(tor --hash-password "$${TOR_AUTH_PASSWORD}" | grep "^16:")
         
         # use /tmp/torrc because the container runs as non-root and cannot write to /etc/tor/
         cat > /tmp/torrc << EOF
         SocksPort 0.0.0.0:$${TOR_SOCKS_PORT:-9050}
         ControlPort 0.0.0.0:$${TOR_CONTROL_PORT:-9051}
-        HashedControlPassword $$HASH
         DataDirectory /var/lib/tor
+        HashedControlPassword $$HASH
```

Comment:
Are you sure this will create no-auth tor? Hash of a empty string is still a hash and it will set password to that hash.